src/Settings/Flavours/Quickest.hs

```diff
@@ -2,13 +2,16 @@ module Settings.Flavours.Quickest (quickestFlavour) where
 
 import Flavour
 import Predicate
+import Oracles.Config.Flag (platformSupportsSharedLibs)
 import {-# SOURCE #-} Settings.Default
 
 quickestFlavour :: Flavour
 quickestFlavour = defaultFlavour
     { name        = "quickest"
     , args        = defaultBuilderArgs <> quickestArgs <> defaultPackageArgs
-    , libraryWays = append [vanilla]
+    , libraryWays = mconcat
+                    [ append [vanilla]
+                    , notStage0 ? platformSupportsSharedLibs ? append [dynamic] ]
```

Comment:
But this is not consistent with the behavior of `quickest` flavor specified in `mk/build.mk`. I remember that dynamic libs will be produced in that one. Will it be a surprise if we redefine this?